pylot/simulation/messages.py

```diff
@@ -87,6 +88,39 @@ def __str__(self):
             self.timestamp, len(self.point_cloud))
 
 
+class IMUMessage(Message):
+    """ Message class to be used to send IMU measurements.
+
+    Attributes:
+        transform: The simulation.utils.Transform of the IMU.
+        acceleration: carla.vector3D linear acceleration measurement in m/s^2
+        gyro: carla.vector3D angular velocity measurement in rad/sec
+        compass: float orientation measurement w.r.t North direction ((0, -1, 0)
+            in Unreal) in radians
+    """
+    def __init__(self, transform, acceleration, gyro, compass, timestamp):
+        """ Initializes the IMU messsage.
+
+        Args:
+            transform: The simulation.utils.Transform of the IMU.
```

Comment:
Can you please do the conversions from Carla transforms and Vector3Ds to Pylot transforms in the driver operator? We want this object to be free of carla arguments so that we can create it from anywhere inside Pylot. Please also update the comments.